sample/sample.py

```diff
@@ -0,0 +1,12 @@
+import sys
+sys.path.append('D:\B\python-jsonpreprocessor')
+
+import JsonPreprocessor
+from pprint import pprint
+
+prepro=JsonPreprocessor.CJsonPreprocessor()
+
+data=prepro.jsonLoad(".\json\json_with_comment.json")
```

Comment:
Hi Son,
important is that this use-case is working.
We assume that most of the users will not require this for robotframework, but for any  other python project.
Thank you,
 Thomas